doc/OnlineDocs/getting_started/index.rst

```diff
@@ -1,9 +1,58 @@
 Getting Started
 ===============
 
-Installation, pyomo command, a simple example, etc
+Installation
+------------
 
-.. doctest::
+Pyomo currently supports the following versions of Python:
+
+* CPython: 2.7, 3.5, 3.6
+
+
+Using CONDA
+~~~~~~~~~~~
+
+We recommend installation with *conda*, which is included with the Anaconda
+distribution of Python.  If you have a different Python distribution, then
+you can install *miniconda* using *pip*:
+
+.. bash::
+    pip install miniconda
+
+You can install Pyomo in your system Python installation
+by executing the following in a shell:
+
+.. bash::
+    conda install -c conda-forge pyomo
+
+Pyomo also has conditional dependencies on a variety of third-party Python packages.  These can also be installed with conda:
```

Comment:
This document should list the third-party packages, as that is relevant to all distributions /package managers (i.e., pip)

Reply:
Perhaps it should.  But I'm pretty sure that no one will maintain this list here.